.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: bionic
+language: c
+jobs:
+  include:
+  - os: linux
+    env: CC=clang CXX=clang++ npm_config_clang=1
+    compiler: clang
+cache:
+  directories:
+  - node_modules
```

Comment:
Isn't it better to always start with a clean state, to replicate users that just downloaded the repo?